### PR TITLE
Fixes #11405: Add a new syntax for rudder parameters

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/11405

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: techniques/system/common/1.0/rudder-parameters.st [hunk 1]
```diff
@@ -18,8 +18,42 @@
...
+#
+
+# The result key is available in the rudder.<namespace> data variable
+# usage example: ${rudder.parameters[my_global_parameter]}
+#
+bundle agent rudder_global_parameters
+{
+  vars:
+    # The files to read
+    "parameters_file" string => "${this.promise_dirname}/../rudder-parameters.json";
```
Comment (ncharles) [inline techniques/system/common/1.0/rudder-parameters.st]: Why not use "${sys.workdir}/inputs" ?

Comment (ncharles) [reply techniques/system/common/1.0/rudder-parameters.st]: actually, this path is wrong, as it would search for the file in common/rudder-parameters.json

**Comment by amousset** [comment]
Commit modified

**Pull request merged**